### DISCUSSION TITLE
Ansible Contributor Summit. Tuesday, April 12, 2022.

### DISCUSSION THREAD

**Discussion by Andersson007** [ansible-collections/community.hashi_vault#233] [Announcements]
##### Ansible Contributor Summit

We are happy to announce that the registration for the [Ansible Contributor Summit](https://hackmd.io/@ansible-community/contrib-summit-202204/%2FlV3o41__RLiX8P2rm-gGKA) is open!

##### Why

- This is a great opportunity for interested people to meet, discuss related topics, share their stories and opinions, get the latest important updates and just to hang out together.

- There will be different announcements & presentations by Community, Core, Cloud, Network, and other teams.

- Current contributors will be happy to share their stories and experience with newcomers.

- There will be links to interactive self-passed instruqt scenarios shared during the event that help newcomers learn different aspects of development.

##### Where/when

Online on Matrix and Youtube. Tuesday, April 12, 2022, 12:00 - 20:00 UTC.

##### How to join

- Add the event to your calendar. Use the [ical URL](https://raw.githubusercontent.com/ansible/community/main/meetings/ical/contribsummit202204.ics) (for example, in Google Calendar "Add other calendars" > "Import from URL") instead of importing the .ics file so that any updates to the event will be reflected in your calendar.

- Check out the [Summit page](https://hackmd.io/@ansible-community/contrib-summit-202204/%2FlV3o41__RLiX8P2rm-gGKA):
  - Add you name to [attendees](https://hackmd.io/@ansible-community/contrib-summit-202204/%2F-JfWR98UTgOEKL7IDIXbKQ).
  - Suggest summit [topics](https://hackmd.io/@ansible-community/contrib-summit-202204/%2FzxOZe_FzR7a9-pw6r6_Ypg) that would be interesting to you to hear about.
  - Vote on and propose changes to [topics](https://hackmd.io/@ansible-community/contrib-summit-202204/%2FzxOZe_FzR7a9-pw6r6_Ypg) suggested by others.
  - If you want to be a presenter, please contact the Ansible Community team via <ansible-community@redhat.com>.
  
We are looking forward to seeing you!:)

**Comment by briantist** [comment]


**Reply by Andersson007** [reply]
@briantist done, thanks!
